mne_bids/config.py

```diff
@@ -136,6 +136,35 @@
            'third dimension (Z) points towards the top of the head.',
 }
 
+REFERENCES = {'mne-bids':
+              'Appelhoff, S., Sanderson, M., Brooks, T., Vliet, M., '
+              'Quentin, R., Holdgraf, C., Chaumon, M., Mikulan, E., '
+              'Tavabi, K., Höchenberger, R., Welke, D., Brunner, C., '
+              'Rockhill, A., Larson, E., Gramfort, A. and Jas, M. (2019). '
+              'MNE-BIDS: Organizing electrophysiological data into the '
+              'BIDS format and facilitating their analysis. Journal of '
+              'Open Source Software 4: (1896).',
```

Comment:
```suggestion
              'Open Source Software 4: (1896). '
              'https://doi.org/10.21105/joss.01896',
```

Adding the DOI is a good idea, because it makes the reference more findable.